### ISSUE TITLE
How to make mask image in efficientdet-seg?

### ISSUE THREAD

**Issue by zmqp111** [google/automl#825]
I am testing segmentation using 'keras/segmentation.py'

The trainning seems to be complete, but when predicting, the mask cannot be created.

How can I make the result a mask image?

Thanks

**Comment by fsx950223** [comment]
Reference https://www.tensorflow.org/tutorials/images/segmentation
